presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReaderOptions.java

```diff
@@ -24,24 +24,28 @@
     private final DataSize maxBlockSize;
     private final boolean mapNullKeysEnabled;
     private final boolean enableTimestampMicroPrecision;
+    private final boolean appendRowNumber;
 
     public OrcRecordReaderOptions(OrcReaderOptions options)
     {
-        this(options.getMaxMergeDistance(), options.getTinyStripeThreshold(), options.getMaxBlockSize(), options.mapNullKeysEnabled(), options.enableTimestampMicroPrecision());
+        this(options.getMaxMergeDistance(), options.getTinyStripeThreshold(), options.getMaxBlockSize(), options.mapNullKeysEnabled(), options.enableTimestampMicroPrecision(),
```

Comment:
presto coding convention is one parameter in one line or all parameters in the same line.